pkg/commands/compute/build.go

```diff
@@ -157,7 +157,7 @@ func (c *BuildCommand) Exec(in io.Reader, out io.Writer) (err error) {
 		return err
 	}
 
-	language, err := language(toolchain, c, out)
+	language, err := language(toolchain, c, in, out, spinner)
```

Comment:
This change was related to simplifying the Toolchain interface's `Build` method (as we already pass a bunch of things into each language constructor, and some of the data was overlapping).